src/Microsoft.Security.Utilities.Core/WellKnownRegexPatterns.cs

```diff
@@ -78,6 +80,7 @@ public static IEnumerable<RegexPattern> HighConfidenceSecurityModelsIterator()
         new AzureApimIdentifiableRepositoryKey(),
         new AzureCacheForRedisIdentifiableKey(),
         new AzureContainerRegistryIdentifiableKey(),
+        // new AzureContainerRegistryLegacyKey(),
```

Comment:
>// new AzureContainerRegistryLegacyKey(), [](http://example.com/codeflow?start=8&length=41)

remove?

Reply:
This needs to be added in. It's commented because the rule is broken.